examples/antd-calendar.js

```diff
@@ -118,6 +118,7 @@ class Demo extends React.Component {
       defaultValue={this.props.defaultCalendarValue}
       showDateInput={state.showDateInput}
       disabledDate={disabledDate}
+      inputMode={"numeric"}
```

Comment:
Needn't pass `inputMode` in example since we should keep example simple.

Reply:
Done